docs/features/insights-cli.md

```diff
@@ -128,10 +128,92 @@ tracked in your Infrastructure-as-Code (IaC) repository.
 You can add the `--delete` flag to the `push rules` command, which
 will delete any Automation Rules from Insights that **do not exist** in your IaC repository. Adding the `--dry-run` flag will explain which Rules would be deleted without making changes to Insights.
 
+### App-Groups
+You can use the Insights CLI to manage the configuration of App-Groups.
+
+Check out the [App-Groups](/features/app-groups) documentation on use cases for configuring App-Groups.
+
+```bash
+insights-cli push app-groups
+```
+
+#### Syncing from Fairwinds Insights
+If you were managing App-Groups via Fairwinds Insights UI. You can download and sync those definitions using the following command:
+
+```
+insights-cli download app-groups
+```
+
+Obs.: folder `app-groups` must exists.
```

Comment:
What does `Obs.` stand for?

Reply:
It was for `Observation`, I will swap to it for a `note`